etc/config-template.toml

```diff
@@ -38,6 +38,9 @@ labels = {}
 [storage]
 # set the path to rocksdb directory.
 data-dir = "/tmp/tikv/store"
+# set the path to raft rocksdb directory, default value is data-dir/raft
+# raft-db-path = "/tmp/tikv/store/raft"
```

Comment:
I don't think it's appropriate to set raft db path here, since storage should not know whether raft is used under the hook.